### ISSUE TITLE
Fix all unit tests.

### ISSUE THREAD

**Issue by airelil** [vasily-v-ryabov/pywinauto-64#28]
Prior to running with other bug fixes or features the all unit tests have to be fixed. It will provide you a solid ground for development of new features and bug fixes. Today if I fix a bug it's hard to see if nothing was broken as unit tests produce too much errors and failures.

**Comment by vasily-v-ryabov** [comment]
You are absolutely right. You've seen pull request from my students about base class ElementInfo. So let me explain. End of May is a deadline for homework. There are 2 groups of students. First one is working on new unit tests, second one is making experiments with new features. Experimental code will be merged into "UIA" branch. So "master" is for UT & bug fixes until 0.5.0 release.

As I understand you're working on unit tests adjustments for found MFC samples (currently for RowList.exe). I will fix ToolTips tests soon, so that 32-bit pass rate achieves almost 100%. Then I'll switch to 64-bit tests fixing. Will let you know about that moment.

BTW, Mark & Michael replied to me. They don't mind about moving official project page & repo to GitHub. Please e-mail to: vasily (dot) v (dot) ryabov (at) gmail (dot) com to discuss some plans privately.

**Comment by airelil** [comment]
Thanks for explanation. One more issue. What about a 32-bit process launched from a 64-bit python ? It won't always work automatically, especially if it involves C-like structure layouts. We could subclass some existing test on 64-bit OS and parameterize them to run with different test apps. Or it's not in the highest priority now?

**Comment by vasily-v-ryabov** [comment]
I thought about some warning or even raising an exception when Python and app process are 32-bit and 64-bit respectively or vice versa. In some cases the root cause of issue is not obvious while it's incorrect Python or app binary.

**Comment by vasily-v-ryabov** [comment]
Different memory layout without switching between Pythons is an ideal scenario, but it needs much more work than I planned for 0.5.0 release. Let's postpone this feature to 0.5.1 or 0.5.2.

**Comment by vasily-v-ryabov** [comment]
OK, I see you've started with different layouts for ListView. I don't mind if you take this commitment for other wrappers as well. For me it's not highest priority yet.
